Quick_Sort/Quick_Sort.dart

```diff
@@ -0,0 +1,101 @@
+import 'dart:io';
+
+class QuickSort {
+    List<int> array = [];
+    QuickSort(this.array);
+
+    void readArray(List<int> a, int n) 
+    {
+        for (int i = 0; i < n; i++) 
+        {
+            print('Add number to array : ');
+            var input = stdin.readLineSync();
+            int x = int.parse(input);
+            assert(x is int);
+            a.add(x);
+        }
+    }
+
+    void printArray(List<int> a) 
+    {
+        for (var i = 0; i < a.length; i++) 
+        {
+            print(a[i]);
+        }
+    }
+
+    int partition(List<int> list, int low, int high) 
+    {
+        int pivot = list[low];
+        int i = high;
+        int temp;
+        for (int j = high; j > low; j--) 
+        {
+            if (list[j] > pivot) {
+            temp = list[j];
+            list[j] = list[i];
+            list[i] = temp;
+            i--;
```

Comment:
Indentation of 35 to 38

Reply:
There is nothing to fix. It is 12 spaces.

Reply:
Should be 16. It is inside if condition.

Reply:
Please fix this

Reply:
Done.